src/util/address.ts

```diff
@@ -0,0 +1,3 @@
+export function truncateAddress(address: string) {
```

Comment:
There's already a `truncate` function in `utils/helpers.ts`, as well as a `<Truncate>` component which shows the full address in a tooltip on hover + includes 'copy to clipboard' functionality - could we use one of those instead?

If the default head / tail values are an issue, we can adjust those but not sure we need a separate function

Reply:
Oh! Totally missed the existing truncate function/component. Yea, I can update.

Reply:
Update https://github.com/regen-network/groups-ui/pull/50/commits/2ec12f5dc11c70fa1256ea90e51f143c5750b690